packages/core/src/lib/markdown-it/highlight/HighlightRule.js

```diff
@@ -10,7 +12,7 @@ class HighlightRule {
   }
 
   static parseRule(ruleString, lineOffset, lines) {
-    const components = ruleString.split('-')
+    const components = ruleString.split(HIGHLIGHT_LINES_RULES_DELIMITER_REGEX)
```

Comment:
As a follow-up to my comment above, if it's better to move the regex to `HighlightRuleComponent.js`, perhaps we can incorporate the splitting part into the `HighlightRuleComponent.parseRuleComponent` method [here](https://github.com/MarkBind/markbind/blob/master/packages/core/src/lib/markdown-it/highlight/HighlightRuleComponent.js#L24-L80) as well? That might make the process more straightforward as parsing the string once using the `HighlightRuleComponent`!